recipe/migrations/lz4_c19.yaml

```
@@ -0,0 +1,8 @@
+migrator_ts: 1652241058
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+
+lz4_c:
+  - '1.9'
```

We could set this to...

```suggestion
  - '1.9.3'
```

...for now and then drop the patch version when we move to `1.10` or `2` whichever is next